README.md

```diff
@@ -23,6 +23,8 @@ For example:
 ```
 brew install go
 brew install dep
+brew install mongo
```

Comment:
I'm not sure if we want/need to prescribe how to run mongo. For example, I run mongo using a Docker container, with port 27018 poked through to localhost.

Reply:
I agree. I think the instructions above about installing and starting Mongo are good enough. We might want to specify the version, though. Production is currently 3.2 while I'm running 3.6.x locally.

Reply:
The same probably goes for the version of Go. How it is installed is irrelevant, I suppose.

OTOH, this is just an example, so maybe we should just leave it.